TestProject1Calculator/AddTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using CSharpCalculator;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TestProject1Calculator
+{
+    [TestClass]
+    internal class AddTests
+    {
+        private Calculator _sut;
+
+        [TestInitialize]
+        public void SetUp()
+        {
+            _sut = new Calculator();
+        }
+
+        [TestCleanup]
+        public void CleanUp()
+        {
+        }
+
+
+        [TestMethod]
+        public void Add_Positive_Success()
```

Comment:
Test's name should be without underscore. And add Test at the end of the test name.